frontend/payment-klarna/pages/ConfirmationPayPalKCO.vue

```diff
@@ -0,0 +1,211 @@
+        <div class="title mt15">
+          <p>{{ i18n.t('Thank you, your order has been purchased completed.') }}</p>
+          <p>{{ message }}</p>
+        </div>
+      </div>
+      <div class="content-detail">
+        <h4>{{ i18n.t('Items Purchased') }}</h4>
+        <div class="items">
+          <!-- eslint-disable-next-line -->
+          <div class="item-detail my10" v-for="item in order.items">
+            <div>{{ item.label }}</div>
+            <div>{{ item.price }}</div>
+          </div>
+        </div>
+      </div>
+      <div class="content-detail">
+        <h4>{{ i18n.t('Order Detail') }}</h4>
+        <div class="total my10 py10">
+          <div>
+            {{ i18n.t('Total') }}
+          </div>
+          <div>
+            {{ order.total }}
+          </div>
+        </div>
+        <!-- eslint-disable-next-line -->
+        <div class="line"></div>
```

Comment:
I think if you do `<div class="line" />` you can remove `eslint-disable-next-line`